Uplift/Controllers/SportsDetailViewController.swift

```diff
@@ -9,23 +9,192 @@
 import UIKit
 
 class SportsDetailViewController: UIViewController {
+     
+    private let collectionView = UICollectionView(frame: .zero, collectionViewLayout: StretchyHeaderLayout())
     
-    private var post: Post
+    private struct Section {
+        var items: [ItemType]
+    }
+    
+    private enum ItemType {
+        case info
+        case players([String])
+        case discussion
+        case comment(Comment)
+        case input
+    }
+    
+    private var post: Post = Post(comment: [], createdAt: Date(), id: -1, userId: -1, title: "", time: "", type: "", location: "", players: 0, gameStatus: GameStatus.open.rawValue)
+    private var section: Section = Section(items: [.info, .players([]), .discussion])
+    private var dropStatus: DropdownStatus = .closed
+    
+    private let sportsDetailHeaderViewReuseIdentifier = "sportsDetailHeaderViewReuseIdentifier"
+    private let infoSectionReuseIdentifier = "infoSectionReuseIdentifier"
+    private let playersSectionReuseIdentifier = "playersSectionReuseIdentifier"
+    private let discussionSectionReuseIdentifier = "discussionSectionReuseIdentifier"
+    private let commentSectionReuseIdentifier = "commentSectionReuseIdentifier"
+    private let inputSectionReuseIdentifier = "inputSectionReuseIdentifier"
```

Comment:
Can we define these reuse identifiers in `Constants.swift -> Identifiers` instead?